pandas-stubs/core/series.pyi

```diff
@@ -260,35 +265,39 @@ class Series(IndexOpsMixin[S1], NDFrame):
         | Interval[_OrderableT]
         | Sequence[Interval[_OrderableT]],
         index: Axes | None = ...,
+        *,
         dtype: Literal["Interval"] = ...,
-        name: Hashable | None = ...,
+        name: Hashable = ...,
         copy: bool = ...,
     ) -> IntervalSeries[_OrderableT]: ...
     @overload
     def __new__(
         cls,
-        data: object | _ListLike | Series[S1] | dict[int, S1] | dict[_str, S1] | None,
-        dtype: type[S1],
+        data: object,
```

Comment:
If we have object here, we need nothing else.

Reply:
Should we consider removing `object` and use `_ListLike | Series[S1] | dict[int, S1] | dict[_str, S1] | None` ?

Reply:
also had to add `Scalar` (or `S1`) as `pd.Series(1)` is valid and tested.